sdk/monitor/Azure.Monitor.Query/src/LogsQueryClient.cs

```diff
@@ -66,14 +68,24 @@ public LogsQueryClient(Uri endpoint, TokenCredential credential) : this(endpoint
         /// <param name="endpoint">The service endpoint to use.</param>
         /// <param name="credential">The <see cref="TokenCredential"/> instance to use for authentication.</param>
         /// <param name="options">The <see cref="LogsQueryClientOptions"/> instance to use as client configuration.</param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
Why are we hiding the constructor?

Reply:
We don't want customers to type in the Uri because they can make a typo. We would rather them use the Audience param under options to set it. So I hid both constructors that take in Uris as parameters